pybamm/models/submodels/pe_degradation/base_phase_transition.py

```diff
@@ -0,0 +1,265 @@
+#
+# Base class for the core-shell model for Positive Electrode (NMC811)
+# phase transition cased degradation, based on paper
+#    Mingzhao Zhuo, Gregory Offer, Monica Marinescu, "Degradation model of
+#    high-nickel positive electrodes: Effects of loss of active material and
+#    cyclable lithium on capacity fade", Journal of Power Sources,
+#    556 (2023): 232461. doi: 10.1016/j.jpowsour.2022.232461.
+#
+import pybamm
+
+
+class BasePhaseTransition(pybamm.BaseSubModel):
```

Comment:
This name will cause problems if anyone wants to add a different positive electrode degradation model, such as acid dissolution. Please change to something like `BasePositiveDegradation` or `BasePositiveElectrodeDegradation`.

Reply:
Good point. How about create a subfolder called "phase transition" inside the pe_degradation folder, and other subfolder like "acid dissolution" etc. pe_degradation is general and can include other non-core-shell submodels.